emission/analysis/config.py

```diff
@@ -2,10 +2,10 @@
 
 def get_config_data():
     try:
-        config_file = open('conf/analysis/debug.conf.json')
+        config_file = open('conf/analysis/debug.conf.json.sample')
     except:
         print("analysis.debug.conf.json not configured, falling back to sample, default configuration")
-        config_file = open('conf/analysis/debug.conf.json.sample')
+        config_file = open('/Users/chuang/Desktop/e-mission-server/conf/analysis/debug.conf.json.sample')
```

Comment:
I am not sure why you need these changes. If you are running your script from the `e-mission-server` directory, they are not required.

Figure out how to set the `pwd` or `cwd` for pycharm if you want to run it from there, or run from the command line.